MetadataExtractor/Extractors/Data/DicomExtract.py

```diff
@@ -0,0 +1,57 @@
+from .IDataExtract import IDataExtract
+from MetadataExtractor.Util import metadataCreation, metadataFormatter
+import pydicom
+from pathlib import Path
+from pydicom._version import __version_info__
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class DicomExtract(IDataExtract):
```

Comment:
Make sure to add `DicomExtract` to the following list to ensure it properly working: https://github.com/BenediktHeinrichs/metadataextractor/blob/main/defaultConfigs.py#L73

Reply:
This has not been resolved